docs/topics/jvm/jvm-records.md

```diff
@@ -0,0 +1,74 @@
+[//]: # (title: Records)
+
+> JVM records feature is [Experimental](components-stability.md). It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-42430).
+>
+{type="warning"}
+
+_Records_ are Java specific classes for storing immutable data. Records carry a fixed set of values – the _records components_.
+As compared to regular Java classes, they have a concise syntax in Java and save you from writing boilerplate code:
+
+```java
+public record Person (String name, int age) {}
+```
+
+The compiler automatically generates a final class inherited from [java.lang.Record](https://download.java.net/java/early_access/jdk16/docs/api/java.base/java/lang/Record.html) with the following members:
+* a private final field for each record component
+* a public constructor
```

Comment:
With parameters for all fields, I suppose? Let's be precise here.

Reply:
Sure, I took your suggestion.